mobile/src/store/courses/actions.ts

```diff
@@ -162,7 +162,8 @@ const becomeMentor = createAsyncThunk<void, void, AsyncThunkConfig>(
         intervieweeUserId: user.id,
         categoryId: course.courseCategoryId,
       };
-      dispatch(interviewsActions.createInterview(payload));
+      await dispatch(interviewsActions.createInterview(payload));
```

Comment:
I think we should prevent next action in case of error
```suggestion
      await dispatch(interviewsActions.createInterview(payload)).unwrap();
```

Reply:
Of course, my bad!